### DISCUSSION TITLE
How do I get other languages at subtitles?

### DISCUSSION THREAD

**Discussion by victorgeel** [openai/whisper#1589] [Show and tell]
My Final results are English only

**Comment by glangford** [comment]
If you have English audio, whisper cannot translate it to other languages.

If you have non-English audio, `--task transcribe` will transcribe the audio into the spoken language. 

> --task {transcribe,translate}
>                         whether to perform X->X speech recognition ('transcribe') or X->English translation ('translate') (default: transcribe)

**Reply by victorgeel** [reply]
I want to change non English to my mother language.
btw I just using by colab pro

**Reply by glangford** [reply]
>  I want to change non English to my mother language.

whisper can only translate non-English to English. Try Google Translate or DeepL.

**Comment by pr0mila** [comment]
Want to fine-tune Whisper for translation tasks for the subtitles? Check out my repo: MediBeng-Whisper-Tiny. It's easy to use and built for efficient training.

https://github.com/pr0mila/MediBeng-Whisper-Tiny
